src/fragmentarium/ui/display/Display.sass

```diff
@@ -14,23 +14,46 @@
       content: "⸣"
 
 @mixin sumerian
-  &--SUMERIAN,
+  &--SUMERIAN
     @include normal-font
     letter-spacing: 0.1em
 
 @mixin emesal
-  &--EMESAL,
+  &--EMESAL
     @include normal-font
     color: pink
     letter-spacing: 0.1em
 
+
+@mixin third-black-border
+  box-shadow: 0 1.3px white,0 2.6px black
```

Comment:
This is very confusing, it is not a border and not third. Why there is even need for a mixin?